src/main/java/frc/robot/subsystems/drivetrain/DriveTrain.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.subsystems.drivetrain;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.kauailabs.navx.frc.AHRS;
+import edu.wpi.first.wpilibj.SPI;
+
+public class Drivetrain extends SubsystemBase {
+    
+    public final AHRS ahrs;
+
+    public Drivetrain(){
```

Comment:
hm i just realized, can't an abstract class still have a constructor? this way updateDrivePowers could be an abstract method